internal/dev_server/ui/src/Flags.tsx

```diff
@@ -247,10 +260,6 @@ function Flags({
             const hasOverride = flagKey in overrides;
             const currentValue = hasOverride ? overrideValue : flagValue;
 
-            if (onlyShowOverrides && !hasOverride) {
```

Comment:
This was causing issues, because we need to not do any filter logic in the display logic for the pagination to work as expected.